webpack/components/extensions/about/SystemStatuses.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { LoadingState } from '../../../move_to_pf/LoadingState';
```

Comment:
I think LoadingState is already in [patternfly-react](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-3/patternfly-react/src/components/LoadingState)